### ISSUE TITLE
Fix the AnnData warning

### ISSUE THREAD

**Issue by nukappa** [rajewsky-lab/spacemake#107]
Maybe depends on `pandas` and might even be ported already.
```
.../dge.py:158: FutureWarning: X.dtype being converted to np.float32 from float64.
In the next version of anndata (0.9) conversion will not be automatic.
Pass dtype explicitly to avoid this warning. Pass `AnnData(X, dtype=X.dtype, ...)` to get the future behavour.
  adata = anndata.AnnData(
```

**Comment by danilexn** [comment]
We don't need either `np.float32` nor `np.float64`, since we are storing integers anyway. We can force to `np.int32`, I don't think single spots or cells will have >2^32 counts

**Comment by danilexn** [comment]
It is fixed in #100, when we merge that PR we can close this
